app/components/create_log_actions_component.html.erb

```diff
@@ -3,7 +3,7 @@
     <% if create_button_href.present? %>
       <%= govuk_button_to create_button_copy, create_button_href, class: "govuk-!-margin-right-6" %>
     <% end %>
-    <% if upload_button_href.present? %>
+    <% if upload_button_href.present? && !user.support? %>
```

Comment:
This button is shown for all users no distinction made between support and non-support.

```
  def upload_button_href
    case log_type
    when "lettings"
      bulk_upload_lettings_log_path(id: "start")
    when "sales"
      bulk_upload_sales_log_path(id: "start")
    end
  end
```

I could change the method to return if support instead of adding `!user.support?`

```
def upload_button_href
    return nil if user.support?

    case log_type
    when "lettings"
      bulk_upload_lettings_log_path(id: "start")
    when "sales"
      bulk_upload_sales_log_path(id: "start")
    end
  end
```

Reply:
Oh, I see, I had forgotten that we were currently seeing the button as support in the general logs list context (just not able to use it unless we were uploading for our own org) - I was confused that a ticket about adding this functionality was doing something that looked more like removing it.

I don't mind where the support user check is in that case, here is fine.